src/data/people.js

```diff
@@ -25,7 +25,7 @@ export default {
     lastName: 'Edwards',
     nickname: 'Gio',
     avatar: './images/giacomo.png',
-    orgID: 'org1',
+    orgID: 'org2',
```

Comment:
You're already using `user5` [in your Storybook story](https://github.com/webex/components/pull/185/files#diff-c43f541e598ddf02dd93bc360731acb3b3b7406d8e8d331ace9b880d7250a21eR34), no need to make another user "external". Please revert changes on `user2`

Reply:
Done.